src/tag/item.rs

```diff
@@ -208,7 +212,10 @@ gen_map!(
 	"TKWD"                  => PodcastKeywords,
 	"COMM"                  => Comment,
 	"TLAN"                  => Language,
-	"USLT"                  => Lyrics
+	"USLT"                  => Lyrics,
+	"MusicBrainz Release Track Id" => MusicBrainzTrackId,
+	"MusicBrainz Album Id"  => MusicBrainzReleaseId,
+	"MusicBrainz Release Group Id" => MusicBrainzReleaseGroupId
```

Comment:
Could you fixup the alignment of the arrows here?

Reply:
That's why I prefer **not** to use any manual alignment. It causes changes in totally unrelated and unchanged lines.